chain/network/src/peer_manager.rs

```diff
@@ -103,6 +121,8 @@ struct ActivePeer {
     connection_established_time: Instant,
     /// Who started connection. Inbound (other) or Outbound (us).
     peer_type: PeerType,
+
+    protocol_version: ProtocolVersion,
```

Comment:
Please add docstring. Is this the protocol version of the peer?

Reply:
Yes. I added docstring and renamed to `peer_protocol_version`.